build.gradle

```diff
@@ -31,7 +31,7 @@ if (System.env.CIRCLE_TEST_REPORTS) {
     test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, getName())
 }
 
-group 'com.palantir.gradle.javadist'
+group 'com.palantir.gradle.sls-packaging'
```

Comment:
note that when this changes we'll need to publish to a different bintray destination and we'll need to get that whitelisted to jcenter.